operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -589,6 +594,10 @@ void addStartupEnvVars(List<V1EnvVar> vars) {
     addEnvVar(vars, ServerEnvVars.SERVER_OUT_IN_POD_LOG, isIncludeServerOutInPodLog());
     addEnvVar(vars, ServerEnvVars.SERVICE_NAME, LegalNames.toServerServiceName(getDomainUid(), getServerName()));
     addEnvVar(vars, ServerEnvVars.AS_SERVICE_NAME, LegalNames.toServerServiceName(getDomainUid(), getAsName()));
+    String dataHome = getDataHome();
```

Comment:
Where is the unit test for this added behavior?

Reply:
Thanks Russ.  Added it for JobStepContext but not for PodStepContext.  Will add unit tests for PodStepContext.

Reply:
@russgold, are you satisfied with the new tests?